test/IntercomUsersTest.php

```diff
@@ -8,7 +10,7 @@
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Middleware;
 
-class IntercomUsersTest extends PHPUnit_Framework_TestCase
+class IntercomUsersTest extends \PHPUnit_Framework_TestCase
```

Comment:
is the `\` actually a php convention 😒

Reply:
Kind of, in a file with a `namespace`, if you don't specify the full path, it will look for the classes relative to the current namespace. That means without the slash it would be looking for `Intercom\Test\PHPUnit_Framework_TestCase` (which of course doesn't exist).

I have added a new commit that imports the external dependency so that we don't need that ugly backslash 😄 

btw, PHPUnit 4 is very EOL, the new versions have a proper namespace (`use PHPUnit\Framework\TestCase`). We should consider upgrading it ⚒